bleach/sanitizer.py

```diff
@@ -593,7 +593,7 @@ def sanitize_css(self, style):
         # the whole thing.
         parts = style.split(';')
         gauntlet = re.compile(
-            r"""^([-/:,#%.'"\sa-zA-Z0-9!]|\w-\w|'[\s\w]+'\s*|"[\s\w]+"|\([\d,%\.\s]+\))*$"""
+            r"""^([-/:,#%.'"\sa-zA-Z0-9!\w]|\w-\w|'[\s\w]+'\s*|"[\s\w]+"|\([\d,%\.\s]+\))*$"""
```

Comment:
I could instead eliminate the whole following lines because it has no corresponding tests:

https://github.com/mozilla/bleach/blob/8d7fd48179b5020d9b1521be7b81e06648d868d3/bleach/sanitizer.py#L590-L601

At this first attempt, I took more conservative path.